README.md

```diff
@@ -228,6 +228,23 @@ This feature was created to support deployments where the user want to use some
 communication like storage network or out of band managment and the virtual functions must exist on boot and not only
 after the operator and config-daemon are running.
 
+#### Disabling SR-IOV Config Daemon plugins
+
+It is possible to disable SR-IOV network operator config daemon plugins in case their operation
+is not needed or un-desirable.
+
+As an example, some plugins perform vendor specific firmware configuration
+to enable SR-IOV (e.g `mellanox` plugin). certain deployment environments may prefer to perform such configuration
+once during node provisioning, while ensuring the configuration will be compatible with any sriov network node policy
+defined for the particular environment. This will reduce or completely eliminate the need for reboot of nodes during SR-IOV
+configurations by the operator.
+
+This can be done by providing the names of plugins to disable as a comma separated list of strings via
+`SRIOV_NETWORK_CONFIG_DAEMON_DISABLE_PLUGINS` environment variable to the operator deployment or using
```

Comment:
nit: I don't know what will be better, maybe `_DISABLED_PLUGINS`

Reply:
i prefer to keep it the same name as the flag (disable-plugins)